news/97.bugfix

```diff
@@ -0,0 +1 @@
+Handle issues that may appear when running multiple times or when plone.volto was installed first. [pbauer]
```

Comment:
This is a bit vague, I'm going to edit it to clarify that it's about the migration.

```suggestion
Make the `migrate_to_volto` process more robust when running multiple times or when plone.volto was installed first. [pbauer]
```

Reply:
thanks